splinepy/spline.py

```diff
@@ -574,6 +574,444 @@ def _get_property(spl, property_):
     return sync_from_core(spl)._data["properties"][property_]
 
 
+class GeometryMapper(SplinepyBase):
```

Comment:
let's move this to a separate module? maybe time to bring `helpme` in real life.